spec.html

```diff
@@ -332,6 +332,21 @@ <h1>Number object</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-terms-and-definitions-bigint-value">
+      <h1>BigInt value</h1>
+      <p>primitive value corresponding to an arbitrary-precision integer value</p>
+    </emu-clause>
+
+    <emu-clause id="sec-terms-and-definitions-bigint-type">
+      <h1>BigInt type</h1>
+      <p>set of all possible BigInt values</p>
+    </emu-clause>
+
+    <emu-clause id="sec-bigint-object">
+      <h1>BigInt object</h1>
+      <p>member of the Object type that is an instance of the standard built-in `BigInt` constructor</p>
```

Comment:
"String object" and "Number object" have NOTEs under them like

> A String object is created by using the String constructor in a new expression, supplying a String value as an argument. The resulting object has an internal slot whose value is the String value. A String object can be coerced to a String value by calling the String constructor as a function (21.1.1.1).

Is there a reason to omit a similar note here?

Reply:
BigInt can't be created via `new`, same as Symbol which doesn't have such note. I'm not sure what text should be in the note for BigInt.